pkg/controllers/resourceapply/resourceapply_test.go

```
@@ -856,6 +856,17 @@ var _ = Describe("applyPodDisruptionBudget", func() {
 				expectModified: true,
 			},
 		),
+		Entry("When there is a mismatch of UnhealthyPodEvictionPolicy it is updated",
+			applyPodDisruptionBudgetArguments{
+				inputFn:     podDisruptionBudget,
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(goimports)